taskcluster/docker/ui-tests/Dockerfile

```diff
@@ -1,6 +1,12 @@
 # %ARG DOCKER_IMAGE_PARENT
 FROM $DOCKER_IMAGE_PARENT
-MAINTAINER Richard Pappalardo <rpappalax@gmail.com>
+
+#----------------------------------------------------------------------------------------------------------------------
```

Comment:
this looks good, but for consistency w/ other Dockerfiles we can drop the METADATA header